src/main.cpp

```diff
@@ -215,11 +215,18 @@ static void frame_handle_failed(void *, struct zwlr_screencopy_frame_v1 *) {
     exit_main_loop = true;
 }
 
+static void frame_handle_damage(void *data,
+    struct zwlr_screencopy_frame_v1 *frame, uint32_t x, uint32_t y,
```

Comment:
Please omit the names of the variables, otherwise we get a bunch of warnings for unused parameters.